packages/grafana-ui/src/components/Menu/Menu.tsx

```diff
@@ -39,7 +39,16 @@ const MenuComp = React.forwardRef<HTMLDivElement, MenuProps>(
         aria-label={ariaLabel}
         onKeyDown={handleKeys}
       >
-        {header && <div className={styles.header}>{header}</div>}
+        {header && (
```

Comment:
this seems unrelated

Reply:
I have no idea why this changed?

Reply:
Maybe I ran prettier?

Reply:
somehow this got pulled into my branch; but i reverted it.